doc/user_manual/runInfo.tex

```diff
@@ -327,6 +327,7 @@ \subsection{RunInfo: Input of Queue Modes}
       in~\cite{MPI}.
       Mode ``MPI''  can either generate a \texttt{qsub} command or can execute
       on selected nodes.
+      Parameters to be given to the mpi command can be specified with the \xmlNode{MPIParam} node.
```

Comment:
Could you provide more descriptions here? In addition, could you add this option to a test?